App/components/listItems/NotificationListItem.tsx

```diff
@@ -7,12 +7,11 @@ import { useTranslation } from 'react-i18next'
 import { StyleSheet, View, Text } from 'react-native'
 import Icon from 'react-native-vector-icons/MaterialIcons'
 
-import { TextTheme, ColorPallet } from '../../theme'
 import { GenericFn } from '../../types/fn'
 import { HomeStackParams, NotificationStackParams, Screens, Stacks } from '../../types/navigators'
 import { parsedSchema } from '../../utils/helpers'
-
-import Button, { ButtonType } from 'components/buttons/Button'
+import { useThemeContext } from '../../utils/themeContext'
+import Button, { ButtonType } from '..//buttons/Button'
```

Comment:
```suggestion
import Button, { ButtonType } from '../buttons/Button'
```
One too many slashes?